rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    @person.rentals << self
+    book.rental << self
+  end
```

Comment:
- [Optional] Dear @Nuga0909, here the logic works fine to use `@book` or just `book` since both references to the same memory location, but since we are creating a rental and the instance variable makes a better case for the association, I would suggest using the `@book.rentals` to add your rentals.